public/css/common.less

```diff
@@ -111,6 +111,10 @@ div.show-more {
     .search-editor-opener,
     .continue-with {
       margin-left: .5em;
+
+      &:last-child {
+        margin-right: -.5em;
```

Comment:
Isn't that the same as https://github.com/Icinga/ipl-web/pull/21/files#diff-89766a82815789bc3a264cb00929ae46f9cc699215193e13e3763ab8eb69ea54R59 ?

Reply:
I think the link target doesn't consider the changes that happened since then.

But how do you like this, then: 0c912205